nat-lab/tests/telio_test.py

```diff
@@ -100,39 +94,36 @@ async def test_set_peer_event(self) -> None:
             # wait for futures to be started
             await asyncio.sleep(0)
             runtime.set_peer(PeerInfo(public_key="AAA", state=State.Connected))
-            await testing.wait_short(future)
+            await future
 
         with pytest.raises(asyncio.TimeoutError):
-            await testing.wait_short(
-                runtime.notify_peer_event("AAA", [State.Connected], [PathType.Relay])
+            await asyncio.wait_for(
+                runtime.notify_peer_event("AAA", [State.Connected], [PathType.Relay]),
+                0.1,
             )
 
     @pytest.mark.asyncio
     async def test_set_derp_state(self) -> None:
         runtime = Runtime()
         runtime.set_derp(create_derpserver_config(State.Connected))
-        await testing.wait_short(
-            runtime.notify_derp_state("1.1.1.1", [State.Connected])
-        )
+        await runtime.notify_derp_state("1.1.1.1", [State.Connected])
 
         # it should pass again
-        await testing.wait_short(
-            runtime.notify_derp_state(
-                "1.1.1.1",
-                [State.Disconnected, State.Connecting, State.Connected],
-            )
+        await runtime.notify_derp_state(
+            "1.1.1.1",
+            [State.Disconnected, State.Connecting, State.Connected],
         )
 
         # it should fail (wrong state)
         with pytest.raises(asyncio.TimeoutError):
-            await testing.wait_short(
-                runtime.notify_derp_state("1.1.1.1", [State.Disconnected])
+            await asyncio.wait_for(
+                runtime.notify_derp_state("1.1.1.1", [State.Disconnected]), 0.1
             )
 
         # it should fail (wrong IP)
         with pytest.raises(asyncio.TimeoutError):
-            await testing.wait_short(
-                runtime.notify_derp_state("1.1.1.2", [State.Connected])
+            await asyncio.wait_for(
+                runtime.notify_derp_state("1.1.1.2", [State.Connected]), 0.1
             )
```

Comment:
I think that `wait_short` is ok, is there any particular reason to replace it with `wait_for`?

Reply:
I just wanted to use that testing.py wrapper